.github/workflows/firebase-functions-merge.yml

```
@@ -13,19 +13,19 @@ on:
       - 'tsconfig.json'
       - 'yarn.lock'
+      - '.github/workflows/firebase-functions-merge.yml'
 jobs:
   deploy_functions_prod:
     runs-on: ubuntu-latest
     environment: production
     steps:
       - uses: actions/checkout@v4
-      - uses: actions/setup-node@v4
+      - name: CI setup action for Firebase deploy
+        uses: ./.github/actions/setup-firebase-deploy
```

actionlint

description is required in metadata of "Setup Node and install Firebase CLI" action at "/inmem/34/4e7789b2-7023-477b-9548-9da77b8115ae/home/jailuser/git/.github/actions/setup-firebase-deploy/action.yml"

(action)